build.gradle

```diff
@@ -33,13 +33,10 @@ sourceCompatibility = 1.8
 targetCompatibility = 1.8
 
 archivesBaseName = modid
-version = minecraft_version + '-' + version
 
 // Fetch build number from Jenkins
 def ENV = System.getenv()
-if (ENV.BUILD_NUMBER) {
-	version = version + "." + "${System.getenv().BUILD_NUMBER}"
-}
+version = version + "." + (ENV.BUILD_NUMBER ?: "local")
```

Comment:
This is the first time you're setting `version`, so just make it `version = (ENV.BUILD_NUMBER ?: "local")` :)

Reply:
actually version will be set by default to the value loaded from gradle.properties

imo not a good practice .. its confusing to guess whatm ight be available in some other file

but i did not want to add a buildSrc project or the like in such a small PR.. until it is necessary